src/components/ProductsTable/ProductsTable.tsx

```diff
@@ -195,7 +208,7 @@ const ProductsTable: FunctionComponent<ProductsTableProps> = ({
               <Td dataLabel={columnNames.name}>
                 <TextContent>
                   <Text component={TextVariants.h3}>
-                    <Link to={`${datum.sku}`}>{datum.name}</Link>
+                    <Link to={`/products/${datum.sku}`}>{datum.name}</Link>
```

Comment:
Suggestion: Was the original link here not working? It seems to work in stage as expected so it shouldn't need to change